package/MDAnalysis/lib/distances.py

```diff
@@ -1318,14 +1523,26 @@ def apply_PBC(coords, box, backend="serial"):
 
 
 def calc_distance(a, b, box=None):
-    """Distance between a and b
+    """Calculates the distance between positions `a` and `b`.
+
+    If the optional argument `box` is supplied, the minimum image convention is
+    applied during distance calculation. Either orthogonal or triclinic boxes
+    are supported.
 
     Parameters
     ----------
-    a, b : numpy.ndarray
-        single coordinate vectors
+    a,b : numpy.ndarray
+        Single position vectors of shape ``(3,)``.
     box : numpy.ndarray, optional
-        simulation box, if given periodic boundary conditions will be applied
+        The unitcell dimensions of the system, which can be orthogonal or
+        triclinic and must be provided in the same format as returned by
+        :attr:`MDAnalysis.coordinates.base.Timestep.dimensions`:\n
+        ``[lx, ly, lz, alpha, beta, gamma]``.
+
+    Returns
+    -------
+    distance : float
+        The distance between positions `a` and `b`.
```

Comment:
Maybe add a 
```reST
See Also
--------
calc_distances : calculate multiple distances efficiently
```

Reply:
Both functions accept both single coordinates and coordinate arrays. There is virtually no difference left between `calc_distance()` and `calc_distances()`. We might want to deprecate `calc_distance()`.